internals/updates/update.go

```diff
@@ -0,0 +1,40 @@
+package updates
+
+import (
+	"context"
+	"time"
+
+	"github.com/rs/zerolog/log"
+)
+
+type GhostUpdaterCleaner interface {
+	Clean(ctx context.Context) error
+}
+
+func Remove(ctx context.Context, updateID int, fn GhostUpdaterCleaner) error {
```

Comment:
`fn` sounds like function. why not use a func?
```suggestion
func Remove(ctx context.Context, cleanFn func(ctx) ) error {
```

then you can call it just 
```updates.Remove(ctx, updaterCleaner.Clean)```

Reply:
for the updateId, may I suggest pass it when initalizing the cleaner? as this is the condition on which you create it?

Reply:
> `fn` sounds like function. why not use a func?
> 
> then you can call it just `updates.Remove(ctx, updaterCleaner.Clean)`

I forgot to change the parameter name "fn" after modifying the function definition from anonymous function to interface, even though my intention was to pass the object.